internal/mining/mining.go

```diff
@@ -154,17 +233,13 @@ const (
 // transaction to be prioritized and track dependencies on other transactions
 // which have not been mined into a block yet.
 type txPrioItem struct {
-	tx       *dcrutil.Tx
-	txType   stake.TxType
-	fee      int64
-	priority float64
-	feePerKB float64
-
-	// dependsOn holds a map of transaction hashes which this one depends
-	// on.  It will only be set when the transaction references other
-	// transactions in the source pool and hence must come after them in
-	// a block.
-	dependsOn map[chainhash.Hash]struct{}
+	tx          *dcrutil.Tx
+	txDesc      *TxDesc
+	txType      stake.TxType
+	fee         int64
+	priority    float64
+	feePerKB    float64
+	resortCount int
```

Comment:
This can be removed now that you removed the associated `resortCount` logic

Reply:
Removed